ZenPacks/zenoss/Microsoft/Windows/datasources/ShellDataSource.py

```diff
@@ -517,7 +522,10 @@ def build_command_line(self, sqlConnection):
         jobs_sqlConnection.append("'|CurrentRunStatus:'$job.CurrentRunStatus;")
         jobs_sqlConnection.append("}}")
         script = "\"& {{{}}}\"".format(
-            ''.join(getSQLAssembly(sqlConnection.version) + sqlConnection.sqlConnection + jobs_sqlConnection))
+            ''.join([BUFFER_SIZE] +
+                    getSQLAssembly(sqlConnection.version) +
+                    sqlConnection.sqlConnection +
+                    jobs_sqlConnection))
```

Comment:
got this while modeling: ERROR zen.PythonClient: Error in zenoss.winrm.WinMSSQL: cannot concatenate 'str' and 'list' objects.